build/nbproject/project.properties

```diff
@@ -80,8 +80,8 @@ javac.classpath=\
     ${file.reference.zap-2.4.3.jar}:\
     ${file.reference.jdom.jar}:\
     ${file.reference.swingx-all-1.6.4.jar}:\
-    ${file.reference.jruby-complete-1.7.4.jar}:\
-    ${file.reference.jython-standalone-2.5.3.jar}:\
+    ${file.reference.jruby-complete-9.1.12.0.jar}:\

```

Comment:
This should be reverted too, for the split?

Reply:
I've already reverted it. I must be doing something wrong with git...

Reply:
Here, have a [look](https://github.com/DarkLighting/zap-extensions/blob/beta/.classpath).

Reply:
Yeap, missed a file there. Fixed.